src/js/actions/DialogActions.js

```diff
@@ -0,0 +1,66 @@
+var AppDispatcher = require("../AppDispatcher");
```

Comment:
https://github.com/airbnb/javascript#22.8

I appologize.  The AppDispatcher is a singleton, so this should be written in PascalCase. No change needed.